addons/dragging/functions/fnc_dropObject_carry.sqf

```diff
@@ -86,3 +86,17 @@ if (_mass != 0) then {
 
 // reset temp direction
 _target setVariable [QGVAR(carryDirection_temp), nil];
+
+// try loading into vehicle
+if (_tryLoad && {!isNull cursorObject} && {([ACE_player, cursorObject, []] call EFUNC(common,canInteractWith))}) then {
+    if (_target isKindOf "CAManBase") then {
+        private _vehicles = [cursorObject, 0, true] call EFUNC(common,nearestVehiclesFreeSeat);
+        if ([cursorObject] isEqualTo _vehicles) then {
+            [ACE_player, _target, cursorObject] call EFUNC(medical_treatment,loadUnit);
```

Comment:
I think we need to check isModLoaded for medical

Reply:
How's this? [438d06b](https://github.com/acemod/ACE3/pull/9138/commits/438d06be19a36702ed5e280579c5acf23a3205c5)